changelog/10553.bugfix.md

```diff
@@ -0,0 +1 @@
+Fixed an over-indent in the Tokenizers section of the Components page of the docs.
```

Comment:
This file should be named `10553.docs.md`

Reply:
done